scripts/generate-markdown/markdown.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path');
+const fs = require('fs');
+const markdownMagic = require('markdown-magic'); // eslint-disable-line import/no-extraneous-dependencies
+
+const header = `| Terra Package      | Version | Status | Dependencies |
+|--------------------|---------|--------|--------------|
+`;
+
+const config = {
+  transforms: {
+    // see: https://github.com/camacho/markdown-magic-subpackage-list
+    SUBPACKAGELIST() {
+      const packagesDir = path.join(process.cwd(), 'packages');
+
+      return header + fs
+        .readdirSync(packagesDir)
+        .map(filename => path.join(packagesDir, filename))
+        .map(dirPath => JSON.parse(fs.readFileSync(path.resolve(dirPath, 'package.json'))))
```

Comment:
whoops, I meant filter if the package.json file exisits. I don't necessarily think we need to read & store each package.json file